mail_outbound_static/README.rst

```diff
@@ -26,8 +26,19 @@ Mail Outbound Static
 |badge1| |badge2| |badge3| |badge4| |badge5| 
 
 This module brings Odoo outbound emails in to strict compliance with RFC-2822
-by allowing for a statically configured From header, with the sender's e-mail
-being appended into the proper Sender header instead.
+by allowing for a dynamically configured From header, with the sender's e-mail
+being appended into the proper Sender header instead. To accomplish this we:
+
+* Add a domain whitelist field in the mail server model.
+* We dinamically select the matching mail server to be use depending on the sender domain:
+
+    1) if mail server is defined when sending then message use this one.
+    2) return the mail server that match with the whitelist domain.
+    3) if not match, return the default mail server available.
```

Comment:
I was able to understand how the module works after reading the help messages of `smtp_from` and `domain_whitelist` fields. These steps are a bit confusing for me. Maybe they could be written taking the cue from those help messages.
What do you think?

Reply:
Hi @eLBati 
Yes, I am totally agree.

I have updated the README based of the help of the fields.
Please could you do a new review of the README?